src/ipc.cpp

```diff
@@ -16,37 +16,39 @@
 
 
 #include "src/ipc.h"
+#include "src/misc/settings.h"
 #include <QDebug>
 #include <QCoreApplication>
+#include <unistd.h>
 
-IPC::IPC() :
-    globalMemory{"qtox"}
+
+IPC::IPC()
+    : globalMemory{"qtox-" GIT_VERSION}
```

Comment:
I do not believe this is an issue. Simply because you cant update a running app on windows and unix users should know better to restart alpha software after update. Anyhow ill make said change.